TS05-Plotting-Time-Series-ggplot-In-R.Rmd

```diff
@@ -425,18 +431,40 @@ spaced to improve readability. If we zoom into the plot, all of the bars are
 black.
 
 <div id="challenge" markdown="1">
-##Challenge
+## Challenge
+Without creating a subsetted dataframelot, plot the precipitation data for 
```

Comment:
Yup - did you get my latest message about the merge? The edits were minor enough, I'll just take care of it.